langchain/scripts/move-cjs-to-dist.js

```diff
@@ -1,38 +1,17 @@
-import { resolve, dirname, parse, format } from "node:path";
```

Comment:
I was hoping to remove this and other scripts - can we not just build these via `package.json` commands?

Reply:
Ah yeah I think that should be possible for them. Let me think on it and I'll get around to it tmro